pkg/inventory/container/container.go

```diff
@@ -31,6 +31,19 @@ func (c *Container) Get(owner meta.Object) (Reconciler, bool) {
 	return p, found
 }
 
+//
+// List all reconcilers.
+func (c *Container) List() []Reconciler {
```

Comment:
This should have been in an earlier PR but didn't make it.  It's needed by virt.